src/pages/Cart.jsx

```diff
@@ -1,51 +1,82 @@
 import React from 'react';
 import '../pages-css/cart.css';
+import { useCart } from '../util_Contexts';
 import CardCart from './CardCart';
 import Navbar from './Navbar';
 
-function Cart() {
+const Cart = () => {
+
+    const { state_Cart } = useCart();
+    const cartItems = state_Cart.cart;
```

Comment:
You can destructure the cart here itself as const { state_Cart : { cart } } = useCart();

Reply:
i will modify it.